app/public/components/chips/indexChip.js

```diff
@@ -16,18 +16,18 @@ import { h } from '/js/src/index.js';
 import { RCT } from '../../config.js';
 const { dataReqParams, pageNames } = RCT;
 
-export default function indexChip(model, index) {
+export default function indexChip(model, pageName, index) {
     const dataPointer = model.getCurrentDataPointer();
     const { page } = dataPointer;
-    const data = model.fetchedData[dataPointer.page][dataPointer.index].payload;
+    const data = model.fetchedData[page][dataPointer.index].payload;
     const { fields } = data;
     const firstField = fields.find((f) => f !== undefined && f.name);
     const targetUrl = `/?page=${page}&index=${index}&${dataReqParams.rowsOnSite}=50&${dataReqParams.site}=1&sorting=-${firstField.name}`;
 
-    return page !== pageNames.periods && model.fetchedData[page][index]
+    return page !== pageNames.periods && model.fetchedData[page][index] && page === pageName
```

Comment:
Names of  variables 'page' and 'pageName' are not sufficiently discriminative against their purpose

Reply:
changed to `currentPage`